core/src/test/java/org/apache/spark/memory/TestMemoryConsumer.java

```diff
@@ -43,6 +45,12 @@ void free(long size) {
     used -= size;
     taskMemoryManager.releaseExecutionMemory(size, this);
   }
+
+  // Exposed for testing
```

Comment:
super nit: we need this comment? This class itself is used for test use only?

Reply:
How about using `@VisibleForTesting` annotation?